README.md

```diff
@@ -41,7 +41,7 @@ steps above.
 
 ## Supported versions
 
-The bindings require dav1d 1.3.0
+The bindings require dav1d>=1.3.0,<=1.5.0
```

Comment:
We don't know the upper bound really. It might still work with 1.6.0, dav1d is not very consistent with mapping breaking API changes to version numbers.

Reply:
Ok, i'm going to write as a warning. My 1.3 version did not work with dav1d 1.4

Reply:
How did it fail?

Reply:
Well, I can't reproduce anymore, but something along the line of system deps failing with `1.4.1`. 

I think that documenting as ( might not work ) would be the best documentation as there is no guarantee that it might still work.